### PR TITLE
CSV download from query results table

### PR DESCRIPTION
**Pull request by jaudick**
This only downloads data the query results table shows. The json data for each row has a lot more data for score and scene information but it wasnt shown on the table so i filtered it out.

### CODE REVIEW

**Comment by rartiss55** [comment]
Yay CSV download -- LGTM!

@brianpippin wondering if we want this merged to master so that people can download their queries for Eval 3.75? (or perhaps we were planning on merging development to master sometime soon anyway)

**Review by rartiss55 — APPROVED** [review]

**Comment by jaudick** [comment]
@brianpippin grouping metrics added in latest commit

File: analysis-ui/src/components/QueryBuilder/queryResultsTable.jsx [hunk 1]
```diff
@@ -199,11 +273,22 @@ class QueryResultsTable extends React.Component {
...
             <div className="query-results-table-holder">
-                <div className="results-group-chooser">
-                    <div className="query-builder-label">Group Table By</div>
-                    <Select className="results-groupby-selector"
-                        onChange={this.setQueryGroupBy} 
-                        options={this.props.groupByOptions}/>
+                <div className="csv-results-group-holder">
+                    <div className="csv-results-child" style={{paddingTop: '25px', paddingLeft: '40px'}}>
+                        <IconButton style={{padding: '7px', borderRadius: '10px', fontSize: '1rem'}} onClick={this.downloadCSV}>
+                            <span class="material-icons">
```
Comment (brianpippin) [inline analysis-ui/src/components/QueryBuilder/queryResultsTable.jsx:279]: This should be className it is throwing an error in the console.

**Review by brianpippin — APPROVED** [review]
I fixed the className and some other JS errors that were in the console, this is good to merge.

**Pull request merged**